postgres/datadog_checks/postgres/postgres.py

```diff
@@ -902,7 +902,7 @@ def _send_database_instance_metadata(self):
                 "collection_interval": self._config.database_instance_collection_interval,
                 'dbms_version': payload_pg_version(self.version),
                 'integration_version': __version__,
-                "tags": self._non_internal_tags,
+                "tags": [t for t in self._non_internal_tags if not t.startswith('db:')],
```

Comment:
Why not use [self.tags_without_db](https://github.com/DataDog/integrations-core/blob/5ae661066490bdae4b41736636193f410a47f63a/postgres/datadog_checks/postgres/postgres.py#L116)?

Reply:
oh derp, didn't realize we defined that in multiple places

Reply:
oh but also, we only want the tags that are _not_ internal

Reply:
too many different combinations 😢